bw2io/ecoinvent.py

```diff
@@ -62,11 +63,11 @@ def pick_a_unit_label_already(obj: dict) -> str:
 def import_ecoinvent_release(
     version: str,
     system_model: str,
-    username: str | None = None,
-    password: str | None = None,
+    username: Union[str, None] = None,
```

Comment:
Please change this, and future instances of this pattern, to use `Optional` instead.